lib/rules/jsx-sort-props.js

```diff
@@ -40,6 +40,59 @@ function isReservedPropName(name, list) {
   return list.indexOf(name) >= 0;
 }
 
+
+function alphabeticalCompare(a, b, ignoreCase) {
+  if (ignoreCase) {
+    a = a.toLowerCase();
+    b = b.toLowerCase();
+  }
+
+  if (a < b) {
+    return -1;
+  } else if (a > b) {
+    return 1;
+  }
+
+  return 0;
+}
+
+function generateFixerFunction(node, context) {
+  var sourceCode = context.getSourceCode();
+  var sortedAttributes = node.attributes.slice(0);
+  var configuration = context.options[0] || {};
+  var ignoreCase = configuration.ignoreCase || false;
+
+  // Sort props according to the context. Only supports ignoreCase.
+  sortedAttributes.sort(function(a, b) {
+    // Put spread attributes at the end.
```

Comment:
autofixers are supposed to always be safe; it's not safe to automatically put spread props on the end because that might change the override order.

In other words, any element that has spread props is simply not safely autofixable - so if we still want to autofix situations without spread props, then we shouldn't be making any changes to elements with spread props.